src/qml/SensorInformationView.qml

```diff
@@ -0,0 +1,63 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import org.qgis 1.0
+import org.qfield 1.0
+import Theme 1.0
+
+Rectangle {
+  id: sensorInformationView
+
+  property int ceilsCount: 4
+  property double rowHeight: 30
+  property color backgroundColor: Theme.mainBackgroundColor
+  property color alternateBackgroundColor: Theme.sensorBackgroundColor
+  property color textColor: Theme.mainTextColor
+
+  width: parent.width
+  height: parent.width > 620
```

Comment:
`620` is recurring in multiple places, would it be worth to put it as a const maybe in Themes.qml?

Reply:
Let me think about it, it's being used in other info panels too. I'll merge this now and figure out way forward for all panels.